applications/HDF5Application/python_scripts/line_output_process.py

```diff
@@ -0,0 +1,79 @@
+from os import stat
+import KratosMultiphysics
+import KratosMultiphysics.HDF5Application as HDF5Application
+from KratosMultiphysics.HDF5Application.point_set_output_process import PointSetOutputProcess
+
+
+def Factory(parameters: KratosMultiphysics.Parameters,
+            model: KratosMultiphysics.Model):
+    if not isinstance(parameters, KratosMultiphysics.Parameters):
+        raise Exception("expecting input parameters of type KratosMultiphysics.Parameters, but got {}".format(type(parameters)))
+    return LineOutputProcess(model, parameters)
```

Comment:
I would return the wrapped process here then you don't have to worry that all the methods are called